Sources/MapboxDirections/RouteStep.swift

```diff
@@ -377,7 +380,7 @@ extension Road: Codable {
             }
             try container.encodeIfPresent(name, forKey: .name)
         } else {
-            try container.encodeIfPresent(ref, forKey: .name)
+            try container.encode(ref ?? "", forKey: .name)
```

Comment:
Had to change that to maintain encoding and decoding to preserve JSON structure as it is reported by Directions.